### ISSUE TITLE
Docker Compose Project Fails to Start with Docker Desktop 4.27

### ISSUE THREAD

**Issue by groogiam** [microsoft/DockerTools#414]
I'm running into an issue where Visual Studio is unable to build my image when running in docker compose in Docker 4.27. Everything was working fine in Docker 4.26.1. I have reported this to Docker https://github.com/docker/for-win/issues/13889. The regular command line docker build commands seem to work so I am unsure if this is something with Docker or some incompatibility with the Docker Tools. Thanks for your help.

**Comment by groogiam** [comment]
This was the error that was in the docker tools pane. It happened during when running the docker compose up command during  the image build for one of the services. I did not capture the entire output and have since rolled back to docker 4.26.1. The issue I referenced above, references several other related tickets for various docker subsystems. To me this looks like something going on with one of the docker subsystems not mounting the context correctly but figured that you all should be aware something is broken that is affecting the Visual Studio tooling.

```
ERROR: failed to calculate checksum of ref 7e5e69fe-0a93-4e6f-94a2-6d69155a6ff7::ry4vsmtlgm8vymwaaeuecfhcn: failed to walk /var/lib/docker/tmp/buildkit-mount1368382612/MyProject/docker-build: lstat /var/lib/docker/tmp/buildkit-mount1368382612/MyProject/docker-build: no such file or directory
```
